spring-components/rdf4j-spring-demo/src/main/java/org/eclipse/rdf4j/spring/demo/ArtDemoCli.java

```diff
@@ -0,0 +1,57 @@
+package org.eclipse.rdf4j.spring.demo;
+
+import java.util.Map;
+import java.util.Set;
+
+import org.eclipse.rdf4j.model.IRI;
+import org.eclipse.rdf4j.spring.demo.model.Artist;
+import org.eclipse.rdf4j.spring.demo.model.Painting;
+import org.eclipse.rdf4j.spring.demo.service.ArtService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class ArtDemoCli implements CommandLineRunner {
```

Comment:
Might be good to put some javadoc on all classes in the demo that briefly explains what it does.